src/main/java/com/corundumstudio/socketio/MultiRoomBroadcastOperations.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2012 Nikita Koksharov
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.corundumstudio.socketio;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+import com.corundumstudio.socketio.protocol.Packet;
+
+/**
+ * Fully thread-safe.
+ *
+ */
+public class MultiRoomBroadcastOperations implements BroadcastOperations {
+
+	private Collection<BroadcastOperations> broadcastOperations;
+
+    public MultiRoomBroadcastOperations( Collection<BroadcastOperations> broadcastOperations ) {
```

Comment:
```suggestion
    public MultiRoomBroadcastOperations(Collection<BroadcastOperations> broadcastOperations) {
```
代码风格不一致吧？ code style